src/components/tag/Tag.tsx

```diff
@@ -20,25 +20,24 @@ export interface TagProps {
 const TagWrapper = styled.div<{ $size: TagSize }>`
   display: inline-flex;
   // default background color (gray)
-  background-color: ${theme.gray200};
-  border-radius: ${theme.radius6};
+  background-color: ${theme.gray100};
+  border-radius: ${theme.radius4};
 
   height: ${p => (p.$size === 'base' ? theme.space20 : theme.space24)};
 `;
 
 const StyledTagLeft = styled.button<Omit<TagProps, 'onClose'>>`
   display: inline-flex;
-  font-size: ${p => (p.size === 'base' ? theme.fontSizeS : theme.fontSizeBase)};
-  background-color: ${theme.gray200};
+  font-size: ${theme.fontSizeS};
+  background-color: ${theme.gray100};
   border-radius: ${theme.radius6} 0 0 ${theme.radius6};
-  gap: ${theme.space8};
   font-weight: normal;
-  padding: 0 ${theme.space8};
-
-  color: ${theme.gray800};
+  padding: ${p => (p.size === 'base' ? '2px 4px' : '4px 4px 4px 8px')};
```

Comment:
If we don't have the variable exist, either to round it up or down so we can utilize the amino variable. Or if it looks really bad, then use the number `px` that you think would look good. It's best to try to use the amino variables that are available, then we would have a unified styling throughout the app.
```suggestion
  padding: ${p => (p.size === 'base' ? `2px ${theme.space4} : `${theme.space4} ${theme.space4} ${theme.space4} ${theme.space8}`)};
```

Reply:
Ill get that updated. I noticed we don't have a 2px variable in Amino yet and I was trying to use it quite a bit. I think thats the only variable we don't have that we use in our designs since we use a 4px grid everywhere and 2px when needed for smaller spacing.

Reply:
That was our original plan for spacing. We might consider using 2px grid if this appears to be used more often in the future.